cmd/oras/internal/option/target.go

```diff
@@ -46,6 +47,11 @@ type Target struct {
 	isOCILayout bool
 }
 
+// EnableDistributionSpecFlag set distribution specification flag as applicable.
+func (opts *Target) EnableDistributionSpecFlag() {
+	opts.applyDistributionSpec = true
+}
```

Comment:
Do we need this as `Target` containers `Remote` and `Remote` has `EnableDistributionSpecFlag()`?